auth-api/tests/unit/services/test_affiliation_invitation.py

```diff
@@ -81,7 +81,7 @@ def test_as_dict(session, auth_mock, keycloak_mock, business_mock, monkeypatch):
             business_identifier=entity_dictionary['business_identifier'])
 
         affiliation_invitation = AffiliationInvitationService.create_affiliation_invitation(affiliation_invitation_info,
-                                                                                            User(user), '')
+                                                                                            User(user), '', 'test')
```

Comment:
We should probably add additional tests that is more focused on testing the multi-environment nature of this change.

The existing updates are good for testing everything is working as previously, but probably not for the scenario it was implemented for, unless I am missing something?